### PR TITLE
Finalize repository tests and middleware, modify the README

### PR DESCRIPTION
**Pull request by dvwhite**
In preparation for leaving the project, I added the following changes:

* Unit tests of the petsRouter.js file
* Modifications to the authId middleware to pass linting rules
* A section in the README explaining the current status of the project

### CODE REVIEW

File: __tests__/routes/pets.test.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+        validation: [],
+        data: {},
+      });
+
+      // The owner id was sent to the end point, and the mocck was called
+      expect(Pets.remove.mock.calls.length).toBe(1);
+      expect(Pets.remove).toHaveBeenCalledWith(pet.id, pet.ownerId);
+    });
+  });
+});
```
Comment (Cberumen51) [inline __tests__/routes/pets.test.js:180]: Great work on all the tests

File: README.md [hunk 2]
```diff
@@ -125,6 +125,72 @@ There are 3 options to get postgresql installed locally [Choose one]:
...
+  [Whimsical Architecture Diagram](https://whimsical.com/architecture-diagram-ADh8unt91rRsvfsWk3YLfA)
+- Add backend middleware, utils or controllers necessary to render data in an easy to use format for the frontend, especially for the `appointments` related features. For instance, one user story reads "As a groomer, I want to be able to see the requests that I get from users and manage them". It would be ideal to structure that data and output it in the desired manner for the front end devs to facilitate development.
+- Add a `favorites` table with saved favorite groomers that the users can access and manage (full CRUD is probably needed)
+
+2. All release 3 features, including those for the following user stories:
+
+- As a client, I want to be able to pay in a convenient and fast way (Stripe)
+- As a groomer, I want to be able to process payments from the customers and receive them in my bank account (Stripe)
+- As a client, I want to be able to rate groomers based on my experience and provide comments
+
```
Comment (Cberumen51) [inline README.md:193]: Beautiful information on the project that was added

**Review by Cberumen51 — APPROVED** [review]
Great additions to the readme of information, tests, and swagger

**Review by hmerritt — APPROVED** [review]
- Exceptionally clear and easy to understand readme.

- Thorough tests for the pets route.

- Nice work adding Swagger documentation for the remaining routes

Great work on all of the above!

**Pull request merged**